protocol.go

```diff
@@ -37,6 +37,7 @@ const (
 const (
 	headerContentType = "Content-Type"
 	headerUserAgent   = "User-Agent"
+	headerTrailer     = "Trailer"
```

Comment:
Figured since this was referenced multiple times, might as well elevate it to a constant.